### DISCUSSION TITLE
Changing Tab Color

### DISCUSSION THREAD

**Discussion by pqant** [furybee/chrome-tab-modifier#383] [Ideas]
Hi, 
I would like to know if it's possible to change the tab colour depending on the URL the user is visiting at that moment. 
Let's say you manage a big portal and there are many articles to follow, but there are some articles that have higher priority. I just want to create a rule for that to change the tab colour so I can easily notice those pages among the others.  ("Tab Modifier" has a similar feature such as changing the page icon.) 

PS: Please don't suggest using the chrome/edge grouping feature etc.. that's not what I was looking for. 

Thanks

**Comment by sebastienfontaine** [comment]
Hi,

Unfortunately, it’s not possible to change the tab color dynamically based on the URL. The only options available are modifying the page icons (as you’ve seen with “Tab Modifier”) or using features like tab grouping in browsers such as Chrome or Edge, even though I understand that’s not what you’re looking for. Those are currently the only feasible ways to visually distinguish certain pages.

Best regards,
